include/oneapi/dpl/algorithm

```diff
@@ -30,6 +30,7 @@
 
 #if _ONEDPL_EXECUTION_POLICIES_DEFINED
 // If <execution> has already been included, pull in implementations
+// include ranges if possible
```

Comment:
Does the comment make sense or we just forgot remove that?

Reply:
I think we forgot to remove it. I'll take care of it.